helix-vcs/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "helix-vcs"
+version = "0.6.0"
+authors = ["Blaž Hrastnik <blaz@mxxn.io>"]
+edition = "2021"
+license = "MPL-2.0"
+categories = ["editor"]
+repository = "https://github.com/helix-editor/helix"
+homepage = "https://helix-editor.com"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+
+git-repository = {version = "0.23.1", default-features = false}
+tokio = { version = "1", features = ["rt", "rt-multi-thread", "time", "sync", "parking_lot", "macros"] }
+similar = "2.2"
+ropey = { version = "1.5", default-features = false, features = ["simd"] }
```

Comment:
You can import `helix-core` and use the re-exported ropey crate so versions aren't specified twice.

Reply:
fixed in latest push